userspace/libsinsp/event.h

```diff
@@ -680,6 +680,10 @@ class SINSP_PUBLIC sinsp_evt {
 
 	inline std::vector<sinsp_evt_param>& get_params() { return m_params; }
 
+	bool has_return_value();
+
+	int64_t get_syscall_return_value();
```

Comment:
Can we inline them?

Reply:
in order to inline them I add to create a new helper method `is_syscall_event`, since we are inlining it we cannot call a not inline method from another cpp file